src/ShiftRegister74HC595.h

```diff
@@ -5,34 +5,34 @@
   Released into the public domain.
 */
 
-#ifndef ShiftRegister74HC595_h
-#define ShiftRegister74HC595_h
+#pragma once
 
 #include "Arduino.h"
```

Comment:
Change to `#include <Arduino.h>` too to be consistent with the `src/ShiftRegister74HC595.cpp` file.